mne/utils.py

```diff
@@ -62,6 +62,18 @@ def nottest(f):
 ###############################################################################
 # RANDOM UTILITIES
 
+
+def _check_copy_dep(inst, copy, kind='inst', default=False):
+    """Check for copy deprecation for 0.13"""
+    if copy is not None:
+        warn('The copy parameter is deprecated and will be removed in 0.13, '
+             'and the behavior will be to operate in-place (like copy=False). '
+             'Use %s.copy() if necessary instead.' % kind, DeprecationWarning)
+    else:
+        copy = default
+    return inst.copy() if copy else inst
```

Comment:
So `_check_copy_dep` is only temporary, and from .13 on, everything will be `inst = inst.copy() if copy else inst` again?